packages/reports/blueprints/navi/index.js

```diff
@@ -22,9 +22,6 @@ module.exports = {
         { name: 'ember-cli-string-helpers',         target: '^1.6.0' },
         { name: 'ember-cp-validations',             target: '3.1.2' },
         { name: 'ember-data-model-fragments',       target: '2.11.5' },
-        { name: 'navi-core',                        target: '^8.0.0' },
```

Comment:
Why are we removing core and data from the reports blueprint?

Reply:
Because version `8.0.0` does not exist.